screens/LoginScreens/SignUpScreen.jsx

```diff
@@ -1,13 +1,119 @@
-import { Text, View, Button } from "react-native";
+import {
+    Text,
+    View,
+    Button,
+    TextInput,
+    KeyboardAvoidingView,
+    ActivityIndicator,
+} from "react-native";
+import React, { useState } from "react";
+import { createUserWithEmailAndPassword, updateProfile } from "firebase/auth";
+import { firebase_auth } from "../../firebase.config.js";
+import FormButton from "../../components/FormButton.jsx";
 
 export default function SignUpScreen({ navigation }) {
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [confirmPassword, setConfirmPassword] = useState("");
+    const [passwordMissmatch, setPasswordMismatch] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const auth = firebase_auth;
+
+    const handleSignUp = async () => {
+        setLoading(true);
+
+        if (passwordMissmatch) {
```

Comment:
PasswordMissmatch is a weird name, something like IsPasswordMatching would be better

Reply:
(A general naming convention for bools like this is is...)